portfolio/src/main/webapp/style.css

```diff
@@ -11,9 +12,99 @@
 body img {
     height: 300px;
     width: 250px;
+    border-radius: 5%;
+    margin: 5%;
 }
 
 #see-more-projects {
     display: none;
     visibility: hidden;
 }
+
+body {
+    background-color: grey;
+    font-family: 'Bellefair';
+    background-image: url('https://media.giphy.com/media/GC7C2Fi902BDG/giphy.gif');
+    /* Set a specific height */
+    height: 500px;
+
+    /* Create the parallax scrolling effect */
+    background-attachment: fixed;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+}
```

Comment:
A general comment about ordering of styles: If you don't have a good reason for ordering or grouping, order the style alphabetically. So, here I would keep the background-styles for the scrolling effect together, but order the other 4 alphabetically.

Reply:
Thanks for letting me know. I did the changes as you suggested :)